### ISSUE TITLE
How to install ?

### ISSUE THREAD

**Issue by ekarudianto** [kamilkp/angular-sortable-view#52]
I think it would be good if you provide the documentation to install this component since I've found nothing inside http://kamilkp.github.io/angular-sortable-view/#?tab=7 or the github page of this component it self.. or did I missed something ?

**Comment by kamilkp** [comment]
Just include `angular-sortable-view` as a dependency in your angular module and use the directives explained in readme.md

**Comment by ekarudianto** [comment]
I see, does this component support the angular version 1.3.15 ? @kamilkp

**Comment by kamilkp** [comment]
Afaik yes. Master

**Comment by ekarudianto** [comment]
so I've been install the component to my angularjs. The thing is, I'm using requirejs but it doesn't work, so I don't really know what I've been missing

I've include the component on my config file

here is the codes

```javascript

require.config(
{
 paths: {
 angular:"path to angular",
 angular-sortable: "path to angular sortable"
            },
 shim: {
   angular-sortable:['angular'] 
 }
}
)

```

and I've load the component 

```javascript

'use strict';

define(
 [
  'angular',
  'angular-sortable'
 ], function(angular) {
   return angular.module('myApp',['angular-sortable-view']);
 }
)

```

this is the html structure

```html
<div class="sortable-container" sv-root sv-part="ui.Treegrid.Options.Header.Jobs">
  <div class="well" ng-repeat="item in ui.Treegrid.Options.Header.Jobs" sv-elements="opts">
  {{item.text}}
  </div>
</div>
```

did I missed something ? @kamilkp

**Comment by kamilkp** [comment]
Requirejs syntax is not supported yet. Feel free to submit a PR for that if you want.

**Comment by BrankoIlic** [comment]
I am new to the angular, and get stuck in using your script, but your answer about including angular-sortable-view as a dependency in angular module, has answered my problems. Thanks!

**Comment by VanTudor** [comment]
I totally agree. I forgot to mention it as a dependency and i wasted like 1 hour trying to figure out why it doesn't work. Thank you for the response, but you should definitely tell people in the read.me how to install it.
